test/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -18,24 +18,31 @@ package connectors
 
 import com.github.tomakehurst.wiremock.client.WireMock._
 import config.AppConfig
+import connectors.CustomsDeclareExportsMovementsConnector._
 import connectors.exchanges.DisassociateDUCRRequest
 import forms.ConsignmentReferences
 import models.requests.{MovementDetailsRequest, MovementRequest, MovementType}
 import org.mockito.BDDMockito._
+import org.scalatest.concurrent.ScalaFutures
 import play.api.http.Status
+import play.api.libs.json.Json
 import play.api.test.Helpers._
+import testdata.CommonTestData.{conversationId, providerId}
+import testdata.MovementsTestData.exampleSubmissionFrontendModel
+import testdata.NotificationTestData.exampleNotificationFrontendModel
 
-class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec {
+class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec with ScalaFutures {
 
   private val config = mock[AppConfig]
+  given(config.customsDeclareExportsMovements).willReturn(downstreamURL)
+
   private val connector = new CustomsDeclareExportsMovementsConnector(config, httpClient)
 
   "Submit Movement" should {
-    given(config.customsDeclareExportsMovements).willReturn(downstreamURL)
 
     "POST to the Back End" in {
       stubFor(
-        post("/movements")
+        post(MovementsSubmissionEndpoint)
```

Comment:
If we change the value of this in the prod code by accident the tests will still pass

I think it was better left as the String as we are asserting the conditions of a contract are met